crates/utils/src/tracing/mod.rs

```diff
@@ -1,6 +1,27 @@
 pub mod grpc;
 
-// Re-export useful traits for open-telemetry traces. This avoids requiring other crates from
-// importing that family of crates directly.
-pub use opentelemetry::trace::Status as OtelStatus;
-pub use tracing_opentelemetry::OpenTelemetrySpanExt;
+use opentelemetry::trace::Status as OtelStatus;
+use opentelemetry::{Key, Value};
```

Comment:
@Mirko-von-Leipzig in the issue you mentioned:
```
We can make set_attribute more ergonomic by adding a secondary trait ToValue which we can then implement for foreign types. i.e. instead of set_attribute("hash", digest.to_hex()), we can just do set_attribute("hash", &digest).
```

I was wondering if this is worthwhile considering that crates can already `impl From<...> for opentelemetry::Value` for their types and rely on the trait as is `, value: impl Into<Value>);`.

WDYT?

Reply:
Most of our core types come from the crates from [`miden-base`](https://github.com/0xPolygonMiden/miden-base/) - and I don't think introducing `open-telemetry` there is a reasonable expectation. Though it could make sense if we really want to standardize on it everywhere.

Reply:
Actually here we go: https://docs.rs/opentelemetry_sdk/latest/opentelemetry_sdk/testing/trace/index.html

They do actually have test infra :)

Reply:
Ah right; yeah this was intended for their 😅 Will cross-post just for traceability.